linalg/blockoperator.hpp

```diff
@@ -237,17 +237,17 @@ class BlockLowerTriangularPreconditioner : public Solver
    //! Add block op in the block-entry (iblock, iblock).
    /**
     * @param iblock  The block will be inserted in location (iblock, iblock).
-    * @param op      The Operator to be inserted.
+    * @param opt     The Operator to be inserted.
     */
-   void SetDiagonalBlock(int iblock, Operator *op);
-   //! Add a block op in the block-entry (iblock, jblock).
+   void SetDiagonalBlock(int iblock, Operator *opt);
```

Comment:
I would prefer to rename the local variable `ops`

Reply:
This is a good suggestion. I think it fits much better.